### ISSUE TITLE
More operating systems support

### ISSUE THREAD

**Issue by Rafii2198** [nrcrast/DbdPerkTool#57]
As EAC got support for Mac OS and Linux, it will be a matter of time when developers enable support it, especially when Steam Deck is coming.

So I think it would be good idea to make this app for more operating system (mostly Linux as I don't know if it is possible to run DBD on Mac OS). And seeing that is Electron based then it would be easy to port it.

**Comment by nrcrast** [comment]
That would be fairly straightforward to do, but I'm not sure it's worth the effort given the playerbase will be so tiny. If I see higher demand I'll do it!

**Comment by Rafii2198** [comment]
Please note that Steam Deck is Linux based meaning it would benefit too ;) And it seems to be doing pretty good. Sadly, there is still no news about anti-cheat support for DBD, all we got is "we are working on it" so I think we would have to wait till that demand will come.

**Comment by nicodemus144** [comment]
agreed. i could use this. i have not been able to get the current version of the Icon Toolbox working under Wine as of yet. would love to have this working natively on Linux, since i find it to be vital to improving the visual appeal of the game.

please do this! thank you!

**Comment by nicodemus144** [comment]
the crash backtrace when i try to run this under Wine.

[backtrace.txt](https://github.com/nrcrast/DbdPerkTool/files/8693204/backtrace.txt)

**Comment by nicodemus144** [comment]
and the renderer.log
[renderer.log](https://github.com/nrcrast/DbdPerkTool/files/8693239/renderer.log)

**Comment by nicodemus144** [comment]
Today DbD on steam is finally working on Linux!

Would be great to be able to use this tool also!

**Comment by Rafii2198** [comment]
> Today DbD on steam is finally working on Linux!

Is it official? Is there any information on this by the devs or is there a chance it shouldn't be here?

**Comment by nicodemus144** [comment]
It hasn't been officially announced that it's working. It has been officially announced that Steam Deck support is coming.

The UI shots from ppl playing on Steam Deck even show custom Steam Deck buttons, so it's being catered to Steam Deck.

Will it still work tomorrow? No idea. But it was working yesterday and I played a few matches. It'll be here at some point, so if you want to support Linux too, no reason to wait. A native Linux version would be great to have!

Thanks!

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
